buildSrc/build.gradle.kts

```diff
@@ -62,15 +62,15 @@ val grGitVersion = "4.1.1"
  * Please check that this value matches one defined in
  *  [io.spine.internal.dependency.Kotlin.version].
  */
-val kotlinVersion = "1.8.10"
+val kotlinVersion = "1.8.22"
```

Comment:
Seems like Kotlin versions here and in `.idea/kotlinc.xml` are different. I don't know it this is a problem, but please inspect.

Reply:
It is. We still cannot use 1.9.0 under `buildSrc` because we have conflict with the one bundled with Gradle. Once we move to Gradle 8.2.x, we'd be able to use Kotlin 1.9.0.